htdocs/themes/xbootstrap/modules/system/blocks/system_block_siteinfo.tpl

```diff
@@ -5,7 +5,7 @@
         <!-- start group loop -->
         <{foreach item=group from=$block.groups}>
             <tr>
-                <th colspan="2"><{$group.name}></th>
+                <th colspan="2"><{$group.name|default:false}></th>
```

Comment:
Should set default for `$group_name` to an empty string since it's being displayed.
```suggestion
                <th colspan="2"><{$group.name|default:''}></th>
```